src/libbson/tests/test-bson-corpus.c

```diff
@@ -19,6 +19,14 @@ skipped_corpus_test_t SKIPPED_CORPUS_TESTS[] = {
     "Unicode and embedded null in code string, empty scope"},
    /* CDRIVER-2223, legacy extended JSON $date syntax uses numbers */
    {"Top-level document validity", "Bad $date (number, not string or hash)"},
+   /* CDRIVER-3500, floating point output differs */
+   {"Double type", "1.2345678921232E+18"},
+   {"Double type", "-1.2345678921232E+18"},
+   /* CDRIVER-4017, libbson does not emit escape sequences */
+   {"Javascript Code", "two-byte UTF-8 (\xc3\xa9)"}, /* \u00e9 */
+   {"Javascript Code", "three-byte UTF-8 (\xe2\x98\x86)"}, /* \u2606 */
+   {"String", "two-byte UTF-8 (\xc3\xa9)"}, /* \u00e9 */
+   {"String", "three-byte UTF-8 (\xe2\x98\x86)"}, /* \u2606 */
```

Comment:
Just changed this to use hex escaping. The original unicode escaping didn't seem to work in MSVC, as the test [was not skipped and failed](https://evergreen.mongodb.com/task_log_raw/mongo_c_driver_windows_2015_release_compile_patch_90f6ffa2db6c489330212ff5571b4897be2a30cd_60c283c69ccd4e1ae864422d_21_06_10_21_27_35/0?type=T#L4459).

This appears to be a can of worms (see: [this repository](https://github.com/jay/compiler_string_test)), so hopefully hex escaping can avoid the problem. Alternatively, we can look into just changing the test itself to no longer use a multi-byte character in its description.